### ISSUE TITLE
Duplicate Clipboard entries added

### ISSUE THREAD

**Issue by savannahar68** [dotenorio/clipboard-manager-electron#24]
If I copy a clip, and again if I copy that same clip, then both the clips are considered as different, and entry is added in-store.

Correct behavior - Only store unique clips and remove duplicate clips if the current clip already exists in DB.

If you want this to be resolved, let me know, I would like to resolve this :)

**Comment by dotenorio** [comment]
This should not happen, I will improve the comparator. Thank you

**Comment by savannahar68** [comment]
Steps to reproduce : 

Consider these 2 clips : 
1. Clip1
2. Clip2

* Copy Clip1 to Clipboard
* Copy Clip2 to Clipboard
* Copy Clip1 again to clipboard => Duplicate entry added
* Copy Clip2 again to clipboard => Duplicate entry added

Keep on doing this, duplicate entries will keep on adding
Check the attached screenshot 

![duplicateClips](https://user-images.githubusercontent.com/20986069/82343904-065b7500-9a11-11ea-990a-673bd3ce35b2.png)


I know the fix, let me know if you want me to resolve this.

**Comment by dotenorio** [comment]
I don't see a problem, it's a how this work...

For clipboard-manager-electron it's not a duplicate registry, it's only a new registry.

And if we think about, this case is uncommon in a normal day.

What you think about that?

**Comment by savannahar68** [comment]
My suggestion and what Window's clipboard history app follows is:

If the Clip copied is same, then remove the previous one, and keep only the latest one(at the top)!

Reason: If I've copied the same clip again, I'll choose the clip which is at top and won't scroll down the list to get the same clip which I've copied earlier. Also if I'm copying over the same clips, again and again, my entire clipboard manager will be filled with same clip, which doesn't look good to eyes.

But the final decision is on you.
